tests/report_examples/Basic_example_vuegen_demo_notebook/quarto_report/quarto_report.qmd

```diff
@@ -0,0 +1,318 @@
+    plot_json = json.load(plot_file)
+
+# Keep only 'data' and 'layout' sections
+plot_json = {key: plot_json[key] for key in plot_json if key in ['data', 'layout']}
+
+# Remove 'frame' section in 'data'
+plot_json['data'] = [{k: v for k, v in entry.items() if k != 'frame'} for entry in plot_json.get('data', [])]
+
+# Convert JSON to string
+plot_json_str = json.dumps(plot_json)
+
+# Create the plotly plot
+fig_plotly = pio.from_json(plot_json_str)
+fig_plotly.update_layout(autosize=False, width=950, height=400, margin=dict(b=50, t=50, l=50, r=50))
+
+fig_plotly.show()
+```
+
+### Pie Plots Biomes Plotly
+```{python}
+#| label: 'Pie Plots Biomes Plotly 4'
+#| fig-cap: ""
+
+with open(report_dir /'../../../../docs/example_data/Basic_example_vuegen_demo_notebook/1_Plots/1_Interactive_plots/4_pie_plots_biomes_plotly.json', 'r') as plot_file:
+    plot_json = json.load(plot_file)
+
+# Keep only 'data' and 'layout' sections
+plot_json = {key: plot_json[key] for key in plot_json if key in ['data', 'layout']}
+
+# Remove 'frame' section in 'data'
+plot_json['data'] = [{k: v for k, v in entry.items() if k != 'frame'} for entry in plot_json.get('data', [])]
+
+# Convert JSON to string
+plot_json_str = json.dumps(plot_json)
+
+# Create the plotly plot
+fig_plotly = pio.from_json(plot_json_str)
+fig_plotly.update_layout(autosize=False, width=950, height=400, margin=dict(b=50, t=50, l=50, r=50))
+
+fig_plotly.show()
+```
+
+### Saline Metagenomics Samples Map Altair
+```{python}
+#| label: 'Saline Metagenomics Samples Map Altair 5'
+#| fig-cap: ""
+
+with open(report_dir /'../../../../docs/example_data/Basic_example_vuegen_demo_notebook/1_Plots/1_Interactive_plots/5_saline_metagenomics_samples_map_altair.json', 'r') as plot_file:
+    plot_json = json.load(plot_file)
+
+# Convert JSON to string
+plot_json_str = json.dumps(plot_json)
+
+# Create the plotly plot
+fig_altair = alt.Chart.from_json(plot_json_str).properties(width=900, height=370)
+
+fig_altair
+```
+
+### Plotly Plot R
+```{python}
+#| label: 'Plotly Plot R 6'
+#| fig-cap: ""
+
+with open(report_dir /'../../../../docs/example_data/Basic_example_vuegen_demo_notebook/1_Plots/1_Interactive_plots/6_plotly_plot_R.json', 'r') as plot_file:
+    plot_json = json.load(plot_file)
+
+# Keep only 'data' and 'layout' sections
+plot_json = {key: plot_json[key] for key in plot_json if key in ['data', 'layout']}
+
+# Remove 'frame' section in 'data'
+plot_json['data'] = [{k: v for k, v in entry.items() if k != 'frame'} for entry in plot_json.get('data', [])]
+
+# Convert JSON to string
+plot_json_str = json.dumps(plot_json)
+
+# Create the plotly plot
+fig_plotly = pio.from_json(plot_json_str)
+fig_plotly.update_layout(autosize=False, width=950, height=400, margin=dict(b=50, t=50, l=50, r=50))
+
+fig_plotly.show()
+```
+
+## Static Plots
+### Number Samples Per Study
+![](../../../../docs/example_data/Basic_example_vuegen_demo_notebook/1_Plots/2_Static_plots/1_number_samples_per_study.png){fig-alt= width=90%}
```

Comment:
so moving the report file around will only work with absolute paths. I did not manage to write proper absolute paths here as it was always prefixed with a dot: `./Users/path/to/image.png`